Cargo.lock

```diff
@@ -3547,6 +3570,12 @@ dependencies = [
  "pin-project-lite",
 ]
 
+[[package]]
+name = "str-buf"
```

Comment:
`str-buf` dependency throws error "error[E0658]: const generics are unstable" on rust 1.50.0.

It was added in https://github.com/kkawakam/rustyline/pull/521

This means minimum Rust version has to be increased for windows builds to merge this, Linux and macOS are not affected.

Reply:
There is an upstream issue: https://github.com/DoumanAsh/str-buf/issues/1

Reply:
Lowered compiler requirement.
Also made it to compile only on windows as it is dependency of `clipboard-win`